server/notification-providers/smtp.js

```diff
@@ -73,7 +113,7 @@ class SMTP extends NotificationProvider {
             bcc: notification.smtpBCC,
             to: notification.smtpTo,
             subject: subject,
-            text: body,
+            html: htmlContent,
```

Comment:
You are throwing away the users template here/ not allowing users to template the part around.